prog-o-meter.py

```diff
@@ -20,25 +25,34 @@
     import tkinter as Tk        # Python >= 3.0
 
 class ProgressGUI(object):
-    
-    """Class contains all things related to the window displaying the prog-o-meter, including text, buttons and actions linked to buttons.
-    
+
+    """Class contains all things related to the window displaying the
+    prog-o-meter, including text, buttons and actions linked to buttons.
+
     Attributes:
         root: a tkinter root.
-        filename: the name of the .txt file storing the user's number of days.
+        filename: the name of the .txt file storing the user's number of

```

Comment:
Does the standard have any approach to line breaks in lists, so that one can easily see where new elements on the "list occurs, i.e. indentation or [...] or something similar to the line below (35), so one can wuickly see it belongs to the line above, and is not a new element to the list of attributes?